docs/config/node.rst

```diff
@@ -255,6 +269,14 @@ Paths
        file:
          file: ${path.logs}/${cluster.name}.log
 
+.. _path.plugins:
+
+**path.plugins**
+  | *Default:* ``plugins``
+  | *Runtime:* ``no``
+
+  Filesystem path to the directory where `plugins`_ are installed.
```

Comment:
Our plugin support was never official and the plugin support is currently not really maintained. I'm not sure if we should add this documentation thus. Maybe we should also remove the related entry inside the `crate.yml` instead.